pom.xml

```diff
@@ -242,27 +242,20 @@
             <dependency>
                 <groupId>net.sf.jasperreports</groupId>
                 <artifactId>jasperreports</artifactId>
-                <version>4.0.1</version>
+                <version>6.1.1</version>
             </dependency>
 
             <dependency>
                 <groupId>org.hibernate</groupId>
                 <artifactId>hibernate-core</artifactId>
-                <version>3.5.4-Final</version>
-                <scope>compile</scope>
-            </dependency>
-
-            <dependency>
-                <groupId>org.hibernate</groupId>
-                <artifactId>hibernate-annotations</artifactId>
-                <version>3.5.4-Final</version>
+                <version>3.6.9.Final</version>
                 <scope>compile</scope>
             </dependency>
 
             <dependency>
                 <groupId>org.hibernate</groupId>
                 <artifactId>hibernate-entitymanager</artifactId>
-                <version>3.5.4-Final</version>
+                <version>3.6.9.Final</version>
```

Comment:
Updating to last versions was not a my goal, I tried to do minimal changes which enough to migrate to java8